lib/matplotlib/testing/decorators.py

```diff
@@ -504,3 +505,44 @@ def _image_directories(func):
     result_dir = Path().resolve() / "result_images" / module_path.stem
     result_dir.mkdir(parents=True, exist_ok=True)
     return baseline_dir, result_dir
+
+
+def generate_matplotlib_baseline_images_from_previous_commit():
+    # Check if current repository is a git repository
+    subprocess.call(['git', '-C', '.', 'status'],
```

Comment:
use cwd=... to run relative to the correct directory.
... <- get from `matplotlib.__file__`